artifact/tests/test_artifact.py

```diff
@@ -37,6 +37,10 @@
 import io
 
 from gem5art import artifact
+from gem5art.artifact._artifactdb import ArtifactDB, getDBConnection
+
+
+_db = getDBConnection(type = 'Mock')
```

Comment:
This should be in the `TestRegisterArtifact` class in `setUp`

Reply:
Added a comment there stating why this is needed